src/main/java/com/jmb/springfactory/controller/api/TaskController.java

```diff
@@ -0,0 +1,74 @@
+package com.jmb.springfactory.controller.api;
+
+import java.util.Date;
+import java.util.List;
+
+import javax.validation.Valid;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import static com.jmb.springfactory.service.UtilsService.*;
+import com.jmb.springfactory.exceptions.NotFoundException;
+import com.jmb.springfactory.exceptions.PersistenceLayerException;
+import com.jmb.springfactory.exceptions.ServiceLayerException;
+import com.jmb.springfactory.model.bo.QueryTaskObject;
+import com.jmb.springfactory.model.dto.TaskDto;
+import com.jmb.springfactory.model.enumeration.PriorityEnum;
+import com.jmb.springfactory.model.enumeration.TaskStatusEnum;
+import com.jmb.springfactory.service.task.TaskService;
+
+@RestController
+@RequestMapping("/schedule/{idSchedule}/order/{idOrder}/task")
+public class TaskController {
+
+  @Autowired
+  private TaskService taskService;
+  
+  @DeleteMapping("/{id}")
+  public void delete(@PathVariable("id") Integer id) {
+    taskService.delete(id);
+  }
+
+  @PostMapping
+  public TaskDto create(@Valid @RequestBody TaskDto task, @PathVariable("idOrder") Integer orderId) 
+      throws ServiceLayerException, NotFoundException, PersistenceLayerException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)